controllers/courses.js

```diff
@@ -883,9 +884,14 @@ router.patch('/:courseId', async (req, res, next) => {
 			req.body.teacherIds.push(currentUserId);
 		}
 		await deleteEventsForCourse(req, res, courseId);
+
+		const requestBody = unarchiveCourse
+			? req.body
+			: { ...req.body, name: req.body.name.trim() };
```

Comment:
```suggestion
		const requestBody =  { ...req.body, name: req.body.name?.trim() };
```
also, could be done the same at line 606